src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class Product {
```

Comment:
Czy ta klasa jest na pewno potrzebna na tym etapie?

Reply:
W sumie na tym etapie to nie jest potrzebna klasa Product , będzie potrzebna podczas mapowania obiektu klasy ProductDto na obiekt klasy Product .

Reply:
Zgadza się, Product to encja, usuń ją proszę i sprawdź CircleCI

Reply:
Nie wiem co zrobić z tym CircleCi :/